functions/zram.bash

```diff
@@ -105,6 +118,15 @@ init_zram_mounts() {
     if ! cond_redirect rm -rf /usr/local/share/zram-config; then echo "FAILED (zram-config share)"; return 1; fi
     if ! cond_redirect rm -rf /usr/local/lib/zram-config; then echo "FAILED (zram-config lib)"; return 1; fi
     if cond_redirect rm -f /etc/logrotate.d/zram-config; then echo "OK"; else echo "FAILED (logrotate)"; return 1; fi
+
+    if [[ -f "$disklistFileDir" ]]; then
+      echo -n "$(timestamp) [openHABian] Removing ZRAM from Amanda local backup... "
+      if cond_redirect sed -i -e '/zram/d' "$disklistFileDir"; then echo "OK"; else echo "FAILED (old config)"; return 1; fi
```

Comment:
will fail to remove ZRAM from AWS if this here already fails

Reply:
What line are you talking about? GitHub's code review system makes it very ambiguous at times.

Reply:
L126, the if test for an AWS config. Won't reach if the previous test fails.

Reply:
I'll look at it, GitHub didn't even show that line in its snippet.

Reply:
Yes it should, there are two possible config files, if the first config file does not exist there is a possibility that the other one will.